src/pages/contributors/vincent-latombe.adoc

```diff
@@ -93,45 +95,48 @@ Jenkins helped with this optimization and provided a way for us to manage all th
 == What have you worked on in Jenkins?
 
 Initially, when I got recruited at CloudBees, they were building proprietary plugins on top of Jenkins as well as contributing to the community.
-It was the very beginning of cloud support, which eventually became CI modern.
-It was basically a prototype, where we were trying to generalize how to provision Jenkins instances automatically without things like Cask, Docker, and a lot of other things that were new at the time.
+When I came in, I started to work on a prototype of a deployment of Jenkins in the cloud.
+We were trying to generalize how some companies were doing massive deployments of Jenkins on clustered systems like Mesos, but packaged in a product that could be run by any of our customers.
+At the time, we started using Mesos and Marathon, while using Docker as a container engine.
+Since we were running on AWS (which was still charging by the hour), and Docker volumes did not exist, we had to create our own system to allocate ebs volumes on the fly.
 
 To focus on my current Jenkins efforts, I've been working on HA support.
 The goal is making a Jenkins installation able to span over multiple machines and horizontally scale.
-This has involved a lot of work, because Jenkins historically had a lot of design principles that were counter to scaling that way.
-I made a lot of patches against Jenkins core in that context, in most cases to make sure any existing implementations would still work.
-It doesn't change that functionality, but opens it in such a way so that we could replace some bits and make them HA-friendly in a corporate context.
-Some of the roadblocks have been pretty interesting over the years, and I try to improve on them so they are no longer blockers.
-
-For instance, the parallel test executor plugin that Kohsuke initially wrote is very convenient when you're looking to parallelize the execution of tests.
-At the time, it was very Java-focused.
-So you run a build with Maven, produce test results, use the JUnit plugin to collect the results, and get a nice report in the Jenkins UI.
-It gets the information from the JUnit plugin about each individual test, such as how much time it took, and then it computes the bucket automatically in the form of `include/exclude` that you can provide back to Maven.
+This has involved a lot of work, because Jenkins historically had a lot of design principles that were counter to scaling that way; a heavy reliance on a file system, holding most of the state in memory.
+We made a lot of patches against Jenkins core in that context.
+In most cases, this involved creating new extension points, to allow users to swap the default implementation for another one that can work in a HA-context.
```

Comment:
```suggestion
This has involved a lot of work because Jenkins historically had a lot of design principles that were counter to scaling that way; a heavy reliance on a file system, holding most of the state in memory.
We made a lot of patches against Jenkins core in that context.
In most cases, this involved creating new extension points, to allow users to swap the default implementation for another one that can work in a HA-context.
```